pallets/asset/src/lib.rs

```diff
@@ -991,7 +991,8 @@ impl<T: Config> AssetFnTrait<T::AccountId, T::Origin> for Module<T> {
             ),
             did,
             None,
-        );
+        )
+        .unwrap();
```

Comment:
Use an `unverified_add_claim` method, since we know that the claim type is valid.